### DISCUSSION TITLE
⨯ Failed to download `Merriweather` from Google Fonts. Using fallback font instead

### DISCUSSION THREAD

**Discussion by alsherif-khalaf** [payloadcms/payload#7233] [Feature Requests & Ideas]
### Payload Version

v3.0.0-beta.67

### Node Version

v20

### Next.js Version

v14.2.5

### Describe the Bug

⨯ Failed to download `Merriweather` from Google Fonts. Using fallback font instead.

How can disable google fonts ?
![image](https://github.com/user-attachments/assets/022a2538-fb39-43d4-a2b4-bd342cf38fbd)
![image](https://github.com/user-attachments/assets/800dad20-8361-4bc9-96d4-eb0d7d719d32)

### Reproduction Steps

Just : npx create-payload-app@beta --use-pnpm

**Comment by jmikrut** [comment]
Hey @alsherif-khalaf — right now, you can't disable the use of Google Fonts, but we could potentially add that as a flag in the future. I imagine that there may have been a temporary outage of Google Fonts which caused this. But otherwise it should not be an issue. 

I will convert this to a Feature Request so we can track it accordingly. If there's enough interest, we will consider the removal of Google Fonts altogether.
